web/auth/interceptors/token_updater.go

```diff
@@ -0,0 +1,79 @@
+package interceptors
+
+import (
+	"context"
+	"sort"
+	"strings"
+	"time"
+
+	pb "github.com/autograde/quickfeed/ag/types"
+	"github.com/autograde/quickfeed/web/auth"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+)
+
+var methods = []string{
```

Comment:
If you make this be a `map[string]struct{}` you can initialize it as:
```go
var methods = map[string]struct{} {
 "UpdateUser": struct{}{},
 "CreateCourse": struct{}{},
...
}
```
Then you don't need to do `sort.Strings(methods)` and `sort.SearchStrings()`... etc